fuzzing/BUILD

```diff
@@ -73,6 +73,22 @@ bool_flag(
     visibility = ["//visibility:public"],
 )
 
+string_flag(
+    name = "compiler_type",
+    build_setting_default = "clang",
+    values = [
+        "clang",
+        "gcc",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
+    name = "is_gcc",
```

Comment:
This could be replaced with the predefined settings in `@rules_cc//cc/compiler`. Then we don't need `compiler_type` at all as the choice will be determined by the C++ toolchain.

Reply:
I originally attempted something based on `@rules_cc//cc/compiler` but got stuck on this error

```
ERROR: .../rules_fuzzing/examples/java/BUILD:31:15: On dependency edge //examples/java:EmptyFuzzTest_bin (3b35bd8) -|binary|-> //examples/java:EmptyFuzzTest_raw_: attempting to transition on '@@rules_cc+//cc/compiler:gcc' which is not a build setting
```

when adding `@rules_cc//cc/compiler:gcc` as an input to the `fuzzing_binary_transition` transition. Do you have an idea how to access the compiler type from the transition without it being an explicit flag?